app/views/external_users/claims/offence_details/_fee_reform_fields.html.haml

```diff
@@ -19,19 +19,16 @@
           = govuk_link_button_secondary('#', class: 'fx-clear-selection') do
             = t('clear_selection', scope: "#{locale_scope}.actions")
 
-.form-group.mod-search-input
-  %label.form-label{ for: 'offence' }
-    = 'Search for the offence'
-    %span.form-hint
-      = 'For example class, band, offence or act name'
+= f.govuk_text_field :offence_search_input,
+  class: 'fx-input',
+  form_group: { classes: 'mod-search-input' },
+  hint: { text: 'For example class, band, offence or act name' },
+  label: { text: 'Search for the offence' }
 
-    %a.fx-clear-search.hidden{ href: '#noop' } Clear search
+= f.hidden_field :offence_id, value: valid_offence ? f.object.offence.id : nil, class: 'fx-model'
+= f.hidden_field :fee_scheme, value: [@claim.fee_scheme.name, @claim.fee_scheme.version], class: 'fx-fee-scheme'
 
-
-    %input#offence.form-control.fx-input{ name: 'offence-search-input', type: 'input', autocomplete: 'off' }
-
-  = f.hidden_field :offence_id, value: valid_offence ? f.object.offence.id : nil, class: 'fx-model'
-  = f.hidden_field :fee_scheme, value: [@claim.fee_scheme.name, @claim.fee_scheme.version], class: 'fx-fee-scheme'
+= govuk_link_button_secondary('Clear search', '#noop', class: 'fx-clear-search hidden')
```

Comment:
It looks as though `fx-clear-search` was used in `app/webpack/javascripts/modules/Modules.OffenceSearchInput.js` but that is no longer the case. Does it still need to be here (and in `Modules.OffenceSearchView.js` on line 149)?

Reply:
it is still referenced at line 17 of that file - to set a handle to this button by the looks of it

Reply:
Are we keeping our custom style for `.fx-clear-search` or are we using the GOVUK defaults?

Reply:
we are keeping it